include/nbl/asset/utils/CSPIRVIntrospector.h

```diff
@@ -648,35 +648,30 @@ class NBL_API2 CSPIRVIntrospector : public core::Uncopyable
 			if (layout)
```

Comment:
Above you need to improve the sanity checks on the `info` and results of other operations being successes